torchaudio/transforms.py

```diff
@@ -1,7 +1,8 @@
 # -*- coding: utf-8 -*-
 
 import math
-from typing import Callable, Optional
+from dataclasses import dataclass, InitVar
```

Comment:
This is causing python 3.6 to fail, since `dataclasses` are from 3.7 onward.

Reply:
We should see if the next release will only support 3.7+. If so, then we don't need to worry about this.

Reply:
there is a python 3.6 [backport](https://pypi.org/project/dataclasses/) of `dataclasses` which is pretty lightweight, we can use it

Reply:
Yeah, that sounds reasonable to me, though I would have preferred avoiding adding a special case for 3.6 :)